frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -785,6 +786,28 @@ function getNoNodeContextMenuOptions(props: NoNodeContextMenuProps): ItemType[]
     );
   };
 
+  const maybeFocusSegment = () => {
+    if (!visibleSegmentationLayer || globalPosition == null) {
+      return;
+    }
+    const clickedSegmentId = getSegmentIdForPosition(globalPosition);
+    const layerName = visibleSegmentationLayer.name;
+    if (clickedSegmentId === 0) {
+      Toast.info("No segment found at the clicked position");
+      return;
+    }
+    const selectedSegmentsOrGroup = state.localSegmentationData[layerName].selectedIds;
+    const additionalCoordinates = state.flycam.additionalCoordinates;
+    const numberOfSelectedSegments = selectedSegmentsOrGroup.segments.length;
+    if (numberOfSelectedSegments < 2) {
```

Comment:
this logic already happens in the new saga (and also doesn't happen when invoking a simple click). therefore, I'd think that it can be removed here, right?